src/_pytest/mark/structures.py

```diff
@@ -467,12 +468,12 @@ def test_function():
 
     # See TYPE_CHECKING above.
     if TYPE_CHECKING:
-        skip = None  # type: _SkipMarkDecorator
-        skipif = None  # type: _SkipifMarkDecorator
-        xfail = None  # type: _XfailMarkDecorator
-        parametrize = None  # type: _ParametrizeMarkDecorator
-        usefixtures = None  # type: _UsefixturesMarkDecorator
-        filterwarnings = None  # type: _FilterwarningsMarkDecorator
+        skip = cast(_SkipMarkDecorator, None)
```

Comment:
Perhaps leave a TODO note to use proper type annotations once we drop Python 3.5?

Reply:
I actually had it in my unsaved editor buffer :facepalm: Pushed now.